smart_campus/app/templates/app/group_edit_page.html

```diff
@@ -0,0 +1,33 @@
+{% extends 'app/base.html' %}
+{% load static %}
+
+{% block stylesheet %}
+  {{ block.super }}
+  <link rel="stylesheet" type="text/css" href="{% static "app/css/edit.css" %}">
+{% endblock %}
+
+{% block content %}
+<div class="ui text container">
+  {% if messages %}
+    {% for message in messages %}
+    <div class="ui info message">
+          {{ message }}
```

Comment:
Indentation.

Reply:
Removed it by the `include` marco template.